clients/cloud/c/README.md

```diff
@@ -0,0 +1,100 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the C client [librdkafka](https://github.com/edenhill/librdkafka).
+
+
+# Prerequisites
+
+* [librdkafka](https://github.com/edenhill/librdkafka) installed on your machine, see [installation instructions](https://github.com/edenhill/librdkafka/blob/master/README.md#instructions).
```

Comment:
The instructions section of the librdkafka README is updated to contain the full installation instructions that were previously in this PR, now in https://github.com/edenhill/librdkafka/pull/2244. It will be merged within the next couple of days, so I think we can merge this examples PR now?

Reply:
👍